networking/p2p/build.gradle

```diff
@@ -7,6 +7,7 @@ dependencies {
   implementation project(':data')
   implementation project(':data:adapter')
   implementation project(':data:metrics')
+  implementation project(':validator:coordinator')
```

Comment:
Can this remain a dependency for tests only?

Reply:
I was unable to quickly create a test support module under validator coordinator (I'm not so familiar with Gradle too much) so I resorted to making it a normal dependency. Maybe we can pair program around this.